client/src/constants/index.ts

```diff
@@ -3,6 +3,11 @@ export const REFRESH_TOKEN_STORAGE_KEY = 'ARKHN_REFRESH_TOKEN';
 export const ID_TOKEN_STORAGE_KEY = 'ARKHN_ID_TOKEN';
 export const TOKEN_DATA_STORAGE_KEY = 'ARKHN_TOKEN_DATA';
 export const STATE_STORAGE_KEY = 'ARKHN_AUTH_STATE';
+export const AUTH_DISABLED =
+  process.env.REACT_APP_AUTH_DISABLED &&
```

Comment:
do you need the `process.env.REACT_APP_AUTH_DISABLED &&` if you use the question mark syntax ?

Reply:
Hum, right, it'll be better!

Reply:
Oh, the argument of `includes` can't be undefined... I can still remove the question mark though

Reply:
right, it's fine :+1: